projects/valtimo/dossier-management/src/lib/components/dossier-management-statuses/dossier-management-status-modal/dossier-management-status-modal.component.html

```diff
@@ -65,7 +65,8 @@ <h3 cdsModalHeaderHeading>
         <input
           cdsText
           formControlName="key"
-          [disabled]="!obs.editingKey"
+          minlength="3"
```

Comment:
I think this should be specified in the form control as a validator instead of in the html

Reply:
Moved